### DISCUSSION TITLE
GenesisException: "Camera not yet supported on Windows" - How can I resolve this?

### DISCUSSION THREAD

**Discussion by vishal-codegemini** [Genesis-Embodied-AI/genesis-world#277] [Q&A] [answered]
**GenesisException**: "Camera not yet supported on Windows" - How can I resolve this?**

I'm encountering an issue while using the Genesis library on my Windows machine. When I attempt to utilize the camera feature, I get the following error:

GenesisException: Camera not yet supported on Windows but is supposed to.

Questions:
What could be causing this issue?
Is this a bug or a configuration problem?
Can someone suggest a solution or workaround?
I’ve already ensured my Python environment is set up correctly and that all dependencies are installed. Any guidance would be greatly appreciated!

**Comment by zhouxian** [comment, marked as answer]
As the message suggested (also suggested in our installation instruction), rendering on windows is not working properly yet... it would be good if someone could help
